src/components/Listing/Listing.tsx

```diff
@@ -1,25 +1,33 @@
 import Image from 'next/image';
-import Link from 'next/link';
-import { BiEditAlt } from 'react-icons/bi';
 import { IoLocationOutline } from 'react-icons/io5';
 import { FaCheck } from 'react-icons/fa';
 import styles from './Listing.module.scss';
 import { IconContext } from 'react-icons';
 import { useRouter } from 'next/router';
 import { getNumberWithSpaces } from '../../utils/getNumberWithSpaces';
+import { useEffect, useState } from 'react';
 
 type Props = {
   data: Listing;
   id: string;
 };
 
 const Listing = ({ data, id }: Props) => {
+  const [isOnLandingPage, setIsOnLandingPage] = useState(false);
+
   const router = useRouter();
 
+  useEffect(() => {
+    const url = router.pathname;
+    if (url === '/') setIsOnLandingPage(true);
+  }, []);
```

Comment:
betoniarzu po co ten useEffect 😭
to wszystko ze statem i useEffectem możesz zrobić w ternery 👍

Reply:
masz rację, cóż za gapa ze mnie.. 😉